front/app/containers/App/index.tsx

```diff
@@ -255,6 +255,30 @@ const App = ({ children }: Props) => {
   const isPagesAndMenuPage = isPage('pages_menu', location.pathname);
   const isInitiativeFormPage = isPage('initiative_form', location.pathname);
   const isIdeaFormPage = isPage('idea_form', location.pathname);
+  const isIdeaShowPage = (urlSegments: string[]) => {
```

Comment:
See comments on the why of this below.

It makes sense to put all these functions in helperUtils. We have similar functions there. But keeping the scope small, since this is a review.